scalableminds/webknossos
# re-enable dnd nml upload in dashboard

This was accidentally lost in #3051 

### URL of deployed dev instance (used for testing):
- https://enableuploadindashboard.webknossos.xyz

### Steps to test:
- try to drag’n’drop an nml file in the Explorative Annotations tab in Dashboard
- should work

### Issues:
- fixes #3073

------
